WMCDuplicateRemover.Tests/WMCDuplicateRemover.Tests.csproj

```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>WMCDuplicateRemover.Tests</RootNamespace>
     <AssemblyName>WMCDuplicateRemover.Tests</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
```

Comment:
Did you gain any particular functionality when you updated to 4.6.1 from 4.0?  Just curious.  I know it's always better to have the latest and greatest, but I was just wondering if you updated to the latest .net framework for any specific reason.